components/resources/quizzes/shared/SkillSelections.tsx

```diff
@@ -37,22 +71,43 @@ const SkillSelections: React.FC<SkillSelectionsProps> = ({
         </div>
         <div className="text-lg font-semibolds px-3">{body}</div>
         <div className="flex flex-col w-full max-w-4xl mx-auto">
-          {selections.map((selection, index) => (
-            <div
-              key={index}
-              className={`flex items-start justify-start w-full px-4 py-2 my-2 cursor-pointer ${
-                selected.includes(selection)
-                  ? "bg-violet-300 text-black-500 border-2 border-black-500 rounded-xl"
-                  : "bg-white text-black-600 border-2 border-black-500 rounded-xl"
-              }`}
-              onClick={() => handleSelection(selection)}
-            >
-              {selections[index]}
-            </div>
-          ))}
+          {selections.map((selection, index) => {
+            const isSelected = currentSelections.includes(selection);
+            const className = [
```

Comment:
Easier and cleaner if you store the tailwind classes in a string. Something like this:

const baseClass = "list of classes here"

selectedClass = isSelected ? 'bg-violet-400' + baseClass : 'bg... + baseClass'

You can also use a string literal

Reply:
Thanks - changes have been implemented!